### ISSUE TITLE
No stdout when compiling

### ISSUE THREAD

**Issue by mgattozzi** [ashleygwilliams/wasm-pack#44]
When using `wasm-pack init` the following output is produced:

```
✅ rustup_add_wasm_target succeeded and stdout was:
✅  cargo_build_wasm succeeded and stdout was:
🏎️ 💨  compiled to wasm!
```

This makes it seem like there was supposed to be output. If there's not it should just cut off the rest of the sentance after and.

**Comment by ashleygwilliams** [comment]
yup, working on this now in combination with #39

**Comment by ashleygwilliams** [comment]
now that i've merged #49 i'm curious if you think this is ok now? i've turned off all the std out shenanigans

**Comment by mgattozzi** [comment]
This looks good to me and is a bit more accurate. I think we can close this in favor of #50
